README.md

```diff
@@ -39,8 +39,8 @@ The agenda for this workshop is as follows:
 | 10:25 AM | [Kustomize](./kustomize/readme.md) |
 | 11:15 AM | Break |
 | 11:25 AM | [Service mesh - Istio ](./ServceMesh/readme.md) |
-| 12:15 PM | [Security](./security/readme.md) |
-| 1:30 PM | End of workshop |
+| 12:15 PM | [Security](./Security/readme.md) |
+| 1:30 PM | [Enterprise Scale for AKS Demo](https://github.com/Azure/caf-terraform-landingzones-starter/tree/starter/enterprise_scale/construction_sets/aks/online/aks_secure_baseline) |
```

Comment:
Workshop is over at 1:30.  Suggest you take this off and reference it somewhere else for attendees to look at on their own time.